lib/code-caller/code-caller-container.js

```diff
@@ -348,11 +354,13 @@ class CodeCallerContainer {
   /**
    * Creates a container and attaches its stdin/stdout/stderr to streams
    * we can write to and read from.
+   *
+   * @param {string} hostDirectory
```

Comment:
It was easier to make TS happy by just passing in a guaranteed non-null directory vs making redundant checks for `this.hostDirectory.path`.